shared-services/src/files/file-service-provider.spec.ts

```diff
@@ -0,0 +1,31 @@
+import { Test, TestingModule } from "@nestjs/testing"
+import { CloudinaryFileService } from "./cloudinary-file.service"
+import { FileServiceProvider } from "./file-service.provider"
+
+// Cypress brings in Chai types for the global expect, but we want to use jest
+// expect here so we need to re-declare it.
+// see: https://github.com/cypress-io/cypress/issues/1319#issuecomment-593500345
+declare const expect: jest.Expect
+
+process.env.CLOUDINARY_CLOUD_NAME = "exygy"
```

Comment:
jw when do we mock `.cloudinaryCloudName` vs `.CLOUDINARY_CLOUD_NAME`? looks like we do both in various tests

Reply:
Thanks for calling this out, it should be cloudinaryCloudName, I forgot to update it in this last place but it seems like it didn't have an impact on the test, since it falls back to an empty string if the value it's looking for is undefined. I first tried reading the value from CLOUDINARY_CLOUD_NAME but when I tested uploading a picture on the partners portal, the value for that variable was an empty string, which means it wasn't being plumbed through properly. I then switched everything to cloudinaryCloudName and cloudinaryUploadPreset and that worked, so I'm using those. I imagine that is because of the exports here https://github.com/metrotranscom/doorway/blob/main/sites/partners/next.config.js